events/bingo/bingoCardApp/bingoCard_app.js

```diff
@@ -17,7 +17,7 @@
     // START UTILITY FUNCTIONS
     // *************************************
 
-    /* ENCODE URL PARAMETERS: Formats data to send to Google sheet*/
+    // Takes a JSON object and translates that into URL query parameters
```

Comment:
Where your name on the header??!? get yo credit!

Reply:
my credit comes from knowing that hundreds of people play Bingo successfully :)